src/amp/lib/permutive.ts

```diff
@@ -0,0 +1,26 @@
+export const generatePermutivePayload = (rawConfig: ConfigType) => {
```

Comment:
It would be really great if this type could be made more explicit. We have a problem in DCR with ConfigType which we're thinking of refactoring so the less it is used the better!

Something like

```typescript
const generatePermutivePayload = ({
   webPublicationDate: string;
   isPaidContent: boolean;
   pageId: string;
   headline: string;
   section: string;
   author: AuthorType; // author in DCR is an object, is it the same here?
   keywords: ?; // not sure what this is
   publishedAt: string;
   edition: Edition;
}) => { ... }
```

would be awesome!